formatters/gocov/example_foobar.out

```diff
@@ -0,0 +1,22 @@
+mode: set
```

Comment:
@safety-jure could you move this file to `formatters/gocov/example/` ?

Reply:
Done. However, the Linux job is failing now for an unrelated reason. Let me know @fede-moya if there is anything else I should do. 

Thank you!

Reply:
Great. I have just opened a PR to fix ci. Merging this one after we solve the ci issue, will ask you to rebase later.  👍🏼